static/js/components/email/AutomaticEmailOptions.js

```diff
@@ -28,15 +25,19 @@ export default class EmailCompositionType extends React.Component {
     </div>
   );
 
+  getEmailType = (sendAutomaticEmails?: boolean): string => (
+    sendAutomaticEmails ? EMAIL_CAMPAIGN : ONE_TIME_EMAIL
+  )
+
   render() {
-    const { automaticEmailType } = this.props;
+    const { sendAutomaticEmails } = this.props;
 
     return (
       <div className="email-type">
         <RadioButtonGroup
           className="type-radio-group"
           name="email-composition-type"
-          valueSelected={automaticEmailType}
+          valueSelected={this.getEmailType(sendAutomaticEmails)}
```

Comment:
I would suggest that here we just use `sendAutomaticEmails` for `valueSelected`, and we use `true` and `false` for the value of the two `RadioButtons` - sorry if I wasn't clear enough, but this is what I was suggesting before.